docs/02-Install.md

```diff
@@ -1,13 +1,111 @@
 # 2.0 - Install
 
-## 2.1 Dependencies
+## 2.1 Dependencies and installation
 
-## 2.2 Setup
+Dependencies and installation information are documented on the LORIS-MRI
+  [README.md](../README.md) file.
 
-## 2.3 Configuration
 
-## 2.4 Pipeline flow
+## 2.2 Configuration
 
-## 2.5 Common errors
+**NEED TO TALK ABOUT THE CREATION OF NIFTI OPTIONS AS IT IS REFERENCED IN THE
+  PIPELINE FLOW DOCUMENTATION.**
 
-[FAQ](AppendixA-FAQ.md)
+## 2.3 Pipeline flow
+
+By default the pipeline is designed for **raw DICOM MRI data**, collected by a
+  longitudinally-organized multisite study with a defined data acquisition
+  protocol. With modifications and further customization, it can handle any
+  modality of imaging data.
+
+Default images insertion into LORIS follows the following steps:
+
+1. Upload of a zip of the DICOM study via the **Imaging Uploader** module (or
+     DICOM study transfer directly on the server hosting LORIS).
+2. DICOM insertion into the `tarchive` tables in order to be able to see the
+     information stored in the DICOMs via the **DICOM Archive** module.
+3. Conversion of the DICOMs into MINC files for protocol validation and
+     insertion into the LORIS database. By default it will also create NIfTI
+     images from the MINC files and insert them into LORIS. See the
+     [Configuration](#2.2-configuration) section above if you want to disable
+     this option.
```

Comment:
Here I will add a small introductory sentence to items a. and b. below:
something like:` One of two possible actions will follow depending on the study-defined protocol:`